app/helpers/permission_helper.rb

```diff
@@ -122,7 +122,7 @@ module PermissionHelper
     ] do |proxy, *args, &block|
       if args[0]
         my_module = args[0]
-        if my_module.active? and my_module.project.active?
+        if my_module.active? and my_module.experiment.project.active?
```

Comment:
Since we will allow experiments to be archived, I believe we should also check for `my_module.experiment.active?` here.